blog/tailcall-n+1-working-2024-08-04.md

```diff
@@ -21,20 +21,20 @@ Unlike a traditional GraphQL implementation where the resolvers are written by h
 
 ## The Algorithm
 
-Tailcall reads your [configuration](../docs/configuration.mdx), parses it, and internally stores it in an efficient graph data-structure that resembles a `HashMap`. This allows `O(1)` access to a GraphQL type which represented as a node by its name. Once the graph data-structure is ready we make it go through a series of validators, one of them being the **N+1 tracker**.
+Tailcall reads your [configuration](https://tailcall.run/docs/configuration.mdx), parses it, and internally stores it in an efficient graph data-structure that resembles a `HashMap`. This allows `O(1)` access to a GraphQL type which represented as a node by its name. Once the graph data-structure is ready we make it go through a series of validators, one of them being the **N+1 tracker**.
 
 :::tip
 To see the actual implementation you can check out the [tracker.rs](https://github.com/tailcallhq/tailcall/blob/main/src/core/config/npo/tracker.rs) implementation.
 :::
 
 We essentially use a Depth-First Search (DFS) algorithm starting at the root query and traversing all the connected nodes. The algorithm works as follows:
 
-1. Initialize a to variables to track the currently traversed path and visited fields so that we can avoid cycles.
+1. Initialize variables to track the currently traversed path and visited fields to avoid cycles.
 2. Start at the root query and begin traversing the graph data structure.
-3. For each field in the current node, check if it has a resolver and is not batched. We know if the node contains a resolver if that node has a [`@http`](../docs/directives.md#http-directive) or a [`@grpc`](../docs/directives.md#grpc-directive).
+3. For each field in the current node, check if it has a resolver and is not batched. We know if the node contains a resolver if that node has a [`@http`](https://tailcall.run/docs/directives.md#http-directive) or a [`@grpc`](https://tailcall.run/docs/directives.md#grpc-directive).
```

Comment:
```suggestion
3. For each field in the current node, check if it has a resolver and is not batched. We know if the node contains a resolver if that node has a [`@http`](/docs/directives.md#http-directive) or a [`@grpc`](/docs/directives.md#grpc-directive).
```

Dont' use absolute URLs.

Reply:
how you are setting the configurations for this conversion 

[WARNING] Blog markdown link couldn't be resolved: (../docs/N+1.md) in source file "/Users/mukeshbishnoi/Desktop/open source/tailcallhq.github.io/blog/tailcall-n+1-working-2024-08-04.md"